tests/models/test_constitutive_laws.py

```diff
@@ -365,7 +371,14 @@ def test_evaluated_values(
             [setup.temperature_variable],
             iterate_index=0,
         )
-    method = getattr(setup, method_name)
+
+    # Obtain the tested method by looping top to bottom through the namespace
+    method_namespace = method_name.split('.')
+    owner = setup
+    for name in method_namespace:
+        method = getattr(owner, name)
+        owner = method
+    # method = getattr(setup, method_name)
```

Comment:
Because of the change from `model.fluid*_*density` to `model.fluid*.*density`: There is 1 more level in the name space.
First owner: `model`, second owner: `fluid`, and only then you come to the method which requires testing according to *tested vs testable methods*